src/platform/forms-system/src/js/web-component-patterns/datePatterns.jsx

```diff
@@ -25,8 +26,11 @@ const currentOrPastDateUI = options => {
   const { title, errorMessages, ...uiOptions } =
     typeof options === 'object' ? options : { title: options };
 
+  const Tag = uiOptions?.useDlWrap ? 'dl' : 'div';
```

Comment:
I was trying to think, is this a choice we want to give? or is there always a correct option here? From when I've used it, its usually because it was meant to fix the review page a11y.

Reply:
I really have no idea. I just added it because it was in ReviewFieldTemplate. I think there is some situation this is used in arrays? But again, not totally sure. If we're letting them pass in uiOptions, then we may want to support this just because uiOptions normally supports it